e2e/app/page/cohort-review-page.ts

```diff
@@ -45,5 +45,17 @@ export default class CohortReviewPage extends AuthenticatedPage {
     await waitWhileLoading(this.page);
     return textContent;
   }
+    /**
+     * Get the participant ID in specified row
+     * @param {number} rowIndex
+     * @return {string} Participant text that was clicked on.
+     */
+   async getParticipantLinkId(rowIndex: number = 1, colIndex: number = 1): Promise<string> {
```

Comment:
nit: I don't think we need `link` text. what do you think of `getParticipantId`?

Reply:
this function was more to validate when on clicking gotonextparticipant the participantid on detailpage matches the one that is in cell(3). I've not used this function yet for the validation.